src/lib/fragments/NglMoleculeViewer.react.js

```diff
@@ -184,6 +184,24 @@ export default class NglMoleculeViewer extends Component {
         }
     }
 
+    // helper function to add molecules to the stage side by side
+    addMolSideBySide(stageObj, sele, stage, chosen, color, xOffset) {
```

Comment:
Hmm, I'm surprised that the existing custom methods haven't showed problems with this already - perhaps it's only strictly necessary when you attach them to the DOM as event handlers, but for safety sake all the methods here that aren't built-in React methods should be explicitly bound to `this` in the `constructor`, like `circos` does for example:

https://github.com/plotly/dash-bio/blob/3010db8dea220daaf3ddaf4d22fa812022a8b4eb/src/lib/fragments/Circos.react.js#L17-L23

Reply:
Fixed in 4c9097c.